ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -187,4 +192,12 @@ public static PathFilter getEventsDirectoryFilter(final FileSystem fs) {
       }
     };
   }
+
+  public static boolean isFirstIncDone(Map<String, String> parameter) {
```

Comment:
All the callers negate this result while using. Why don't we change it to isFirstIncPending? It will make it uniform with stored DB parameter name as well.

Reply:
done